src/groupWith.js

```diff
@@ -0,0 +1,33 @@
+var _curry2 = require('./internal/_curry2');
+
+/**
+ * Takes a list and returns a list of lists where each sublist contains only equal elements.
```

Comment:
It's incorrect to write "equal elements" here, since the grouping function is provided by the caller.